pegasus/helpers/section_api_helpers.rb

```diff
@@ -1,14 +1,11 @@
 require 'cdo/user_helpers'
-# TODO -- remove this and change the APIs below to check logged in user instead of passing in a user id
-module Dashboard
-  def self.admin?(user_id)
-    !!DASHBOARD_DB[:users].where(id: user_id, admin: true).first
-  end
-end
+require_relative '../helper_modules/dashboard'
 
+# TODO -- change the APIs below to check logged in user instead of passing in a user id
 class DashboardStudent
+
   def self.fetch_user_students(user_id)
-    DASHBOARD_DB[:users].
+    Dashboard::db[:users].
```

Comment:
This `DASHBOARD_DB` => `Dashboard::db` change everywhere allows our tests to do the following:

```ruby
Dashboard.stubs(:db).returns(fake_db)
```